errors.json

```diff
@@ -175,165 +164,141 @@
 		"message": "File content is invalid.",
 		"http": 400,
 		"tags": [
-			"File Management",
-			"Processes"
+			"File Management"
 		]
 	},
 	"FileLocked": {
 		"description": "The file is locked by a running job or another process.",
-		"message": "File '{file}' is locked.",
+		"message": "File '{file}' is locked by another process.",
 		"http": 400,
 		"tags": [
 			"File Management"
 		]
 	},
 	"ProcessGraphNotFound": {
-		"description": "The requested user-defined process does not exist.",
+		"description": "The requested user-defined process does not exist. To be used for all endpoints starting with `/process_graphs`.",
 		"message": "User-defined process '{identifier}' does not exist.",
 		"http": 404,
 		"tags": [
-			"Process Graph Management"
+			"User-Defined Processes"
 		]
 	},
 	"ProcessGraphMissing": {
```

Comment:
`ProcessGraphNotFound` us for requests where /process_graphs/{id} has an non-existing id given (that's why the HTTP code is 404).

`ProcessGraphMissing` is for the case where you don't specify the `process` property in the JSON body of POST/PUT requests to /jobs, /services etc. It could indeed be captured by `ProcessParameterInvalid`. Leaving this open to the implementer and leaving the old message in for backward-compatibility.